Classes/UIAutomationHelper.m

```diff
@@ -177,7 +177,7 @@ - (void)deactivateAppForDuration:(NSNumber *)duration {
         NSAssert([[[[self target] frontMostApp] name] isEqual:@"SpringBoard"], @"If reactivation is failing, the app is likely still open to SpringBoard.");
         
         // Tap slightly above the middle of the screen, otherwise it doesn't resume on an iPad Pro
-        [[[self target] frontMostApp] tapWithOptions:@{@"tapOffset": @{@"x": @(.5), @"y": @(.36)}}];
+        [[[self target] frontMostApp] tapWithOptions:@{@"tapOffset": @{@"x": @(.7), @"y": @(.36)}}];
```

Comment:
Guessing that this change might possibly break tests for folks. I'd explicitly call this out in the commit message as a behavior change.

Reply:
I tried this on simulators starting on iOS 10+ and it worked on all the different versions. I can call this out though just to make sure